CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing guidelines
+
+We thank you in advance :thumbsup: :tada: for taking the time to contribute, whether with *code* or with *ideas*, to the project.
+
+
+## Did you find a bug?
```

Comment:
As @greysaruman  suggested, it would be great to have an example of a real bug in this section.

Reply:
I added an issue template in #16.
I keept a standard one from github, feel free to suggest any update.